R/safe2x2Test.R

```diff
@@ -1550,19 +1556,50 @@ simulateWorstCaseQuantileTwoProportions <- function(na, nb, priorValues,
     currentPower <- mean(stopEs >= 1/alpha)
 
     #we look for the worst case (1-beta)% stopping time or power: store only that one
+    #also store the standard deviation, obtained through bootstrapping
+    #note that we do this only if we have found a new worst case: omit bootstrapping for every
+    #distribution.
     #also store the implied target belonging to the worst case power: exp(Expected [log E-waarde])
     if (currentQuantile >= currentWorstCaseQuantile) {
       currentWorstCaseQuantile <- currentQuantile
+      if (expectedStopTime) {
+        bootResult <- boot::boot(
+          data = stoppingTimes,
+          statistic = function(x, idx) {
+            mean(x[idx])
+          },
+          R = bootN
+        )
+      } else {
+        bootResult <- boot::boot(
+          data = stoppingTimes,
+          statistic = function(x, idx) {
+            quantile(x[idx], 1 - beta)
+          },
+          R = bootN
+        )
+      }
+      worstCaseQuantileTwoSe <- sd(bootResult[["t"]])
     }
     if (currentPower <= currentWorstCasePower) {
       currentWorstCasePower <- currentPower
+      bootResultPower <- boot::boot(
```

Comment:
I'll write a generic wrapper for this. Add a todo here so it's easier to swap. I'm not sure why the bootstrap is run while you run through the loop though.